digits/model/tasks/torch_train.py

```diff
@@ -136,6 +135,8 @@ def task_arguments(self, resources, env):
                 '--save=%s' % self.job_dir,
                 '--snapshotPrefix=%s' % self.snapshot_prefix,
                 '--snapshotInterval=%s' % self.snapshot_interval,
+                '--mean=%s' % self.dataset.path(constants.MEAN_FILE_IMAGE),
+                '--labels=%s' % self.dataset.path(self.dataset.labels_file),
```

Comment:
Why are you adding labels at all? [It's already being done below](https://github.com/NVIDIA/DIGITS/blob/13d28b4079eead5e9a3b2afadf1806d3f0af0834/digits/model/tasks/torch_train.py#L150).